src/app.js

```diff
@@ -6,8 +6,75 @@ import _ from 'underscore';
 import './css/foundation.css';
 import './css/style.css';
 
-console.log('it loaded!');
+//models and collections
+import Trip from './models/trip';
+import TripList from './collections/trip_list';
+
+
+const tripList = new TripList();
+
+let tripTemplate;
+let tripInfoTemplate;
+const render = function(tripList) {
+  console.log('in render');
+  $('#trip-list').append('<tr><th>ID</th><th>CONTINENT</th><th>COST</th><th>CATEGORY</th></tr>');
+  tripList.forEach((trip) => {
+    $('#trip-list').append(tripTemplate(trip.toJSON()));
+  });
+
+const hideTripDetailsButton = function hideTripDetailsButton() {
+  $('#hide-trip-details').click(function(event) {
+    $('#trip-list').removeClass("small-6");
+    $('#trip-list').addClass("small-11");
+    event.preventDefault();
+    $('#trip-details').hide();
+  });
+  };
+
+  $('tr').click(function() {
+    console.log('inside click');
+    const selectedId = parseInt($(this)[0].id);
+    console.log(selectedId);
+    const url = `http://ada-backtrek-api.herokuapp.com/trips/${selectedId}`;
+    const selectedTrip = tripList.findWhere({id: selectedId});
+    console.log(url);
+    $('#trip-details').show();
+    $.get(url, function(response) {
```

Comment:
We should try to use Backbone's `fetch` function for models here instead of directly using jQuery's ajax functions.